Makefile

```diff
@@ -303,6 +305,22 @@ static-checks:
 fix:
 	goimports -w $(SRC_FILES)
 
+.PHONY: format-check
+format-check:
+	@$(CONTAINERIZED) \
+	sh -c '$(GIT_CONFIG_SSH) \
+	files=$$(gofmt -l ./pkg ./controllers ./api ./test); \
+	[ "$$files" = "" ] && exit 0; \
+	echo The following files need a format update:; \
+	echo $$files; \
+	echo Try running \"make fmt\" and committing any changes; \
```

Comment:
I think in other repos we prefer using goimports. Could we use that instead?

Reply:
Those aren't the same things it seems. I tried running goimports on the operator and it changed several imports and they looked incorrect.

Reply:
Perhaps a difference in go version between containerized and a local non-containerized builds?

Reply:
To clarify, I did expect that goimports would do the work of `go fmt` as well as clean up imports. However if the imports look wrong, then my above comment could be one reason for this behavior.

Reply:
Ok yeah looks like if I containerize and use go-build to run goimports (which already exists under the `fix` make target) it doesn't change a bunch of stuff incorrectly.

Reply:
Is it ok using the target we already had that is called `fix` which runs goimport?

Reply:
Awesome. I think that works.